.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] "
+labels: bug
+assignees: ''
+
+---
+
+**Is this a new bug?**
+In other words: Is this an error, flaw, failure or fault? Please search issues to see if someone has already reported the bug you encountered. 
+
+If this is a request for help or troubleshooting code in your own Pinecone project, please join the [Pinecone Community Forum](https://community.pinecone.io/).
+
+- [ ] I believe this is a new bug
+- [ ] I have searched the existing issues, and I could not find an existing issue for this bug
+
+**Describe the bug**
+Describe the functionality that was working before but is broken now.
+
+**Error information**
+If you have one, please include the full stack trace here. If not, please share as much as you can about the error.
+
+**Steps to reproduce the issue locally**
+Include steps to reproduce the issue here. If you have sample code or a script that can be used to replicate this issue, please include that as well (including any dependent files to run the code).  
+
+**Environment**
```

Comment:
I'd add here that it'd be helpful to know what operating system they're working on